### DISCUSSION TITLE
HDhomerun TV guide from HDhomerun api

### DISCUSSION THREAD

**Discussion by Darin755** [jellyfin/jellyfin-meta#63] [Ideas]
Currently Jellyfin is not able to take advantage of the free TV guide provided by HDhomerun. This guide comes from the HDhomerun servers and is accessible though a access key unique to each device. The returned value from the API is in a JSON format.

In a perfect world you wouldn't need to bother with XML guides because jellyfin could automatically pull the latest guide from the HDhomerun api.

I am learning web development but I don't know C# so I would need someone else's help implementing this. I figured this would be a good place to see if the rest of the community would be interested.

**Comment by thornbill** [comment]
Yeah this would be great! You might want to post it on https://features.jellyfin.org/ also if it is not already there.
